src-docs/src/services/playground/knobs.js

```diff
@@ -0,0 +1,352 @@
+
+      let valueKey = val || defaultValue;
+
+      if (numberOfOptions < 1) {
+        if (valueKey && !valueKey.includes('__')) {
+          valueKey = `${valueKey}__${name}`;
+        }
+        const flattenedOptions = optionsKeys.map(key => ({
+          id: `${key}__${name}`,
+          label: options[key],
+        }));
+
+        return (
+          <Spacing>
+            {/* <Label tooltip={getTooltip(description, type, name)}>{name}</Label> */}
+
+            <EuiRadioGroup
+              options={flattenedOptions}
+              idSelected={valueKey}
+              onChange={id => {
+                let val = id;
+                if (val.includes('__')) val = val.split('__')[0];
+                globalSet(val);
+              }}
+              name={`Select ${name}`}
+            />
+            {error && <div>error {error}</div>}
+          </Spacing>
+        );
+      } else {
+        const flattenedOptions = optionsKeys.map(key => ({
+          value: key,
+          text: options[key],
+        }));
+
+        return (
+          <Spacing>
+            {/* <Label tooltip={getTooltip(description, type, name)}>{name}</Label> */}
+
+            <EuiSelect
+              fullWidth
+              id={name}
+              options={flattenedOptions}
+              value={valueKey}
+              onChange={e => {
+                globalSet(e.target.value);
+              }}
+              aria-label={`Select ${name}`}
+              compressed
+            />
+            {error && <div>error {error}</div>}
+          </Spacing>
+        );
+      }
+
+    case PropTypes.ReactNode:
+    case PropTypes.Function:
+    case PropTypes.Array:
+    case PropTypes.Object:
+    // return (
+    //   <Spacing>
+    //     {/* <Label tooltip={getTooltip(description, type, name)}>{name}</Label> */}
+    //     <EuiTextArea
+    //       placeholder={placeholder}
+    //       value={val ? val : ''}
+    //       onChange={e => {
+    //         globalSet(e.target.value);
+    //       }}
+    //     />
+    //     {error && <div>error {error}</div>}
+    //   </Spacing>
+    // );
+
+    case PropTypes.Custom:
+      return null;
+    default:
+      console.log('name', name);
+      return assertUnreachable();
+  }
+};
+
+const KnobColumn = ({ state, knobNames, error, set }) => {
+  return (
+    <>
+      {knobNames.map((name, idx) => {
+        let humanizedType = '';
+
+        if (
+          state[name].custom &&
+          state[name].custom.origin &&
+          state[name].custom.origin.type
+        )
+          humanizedType = humanizeType(state[name].custom.origin.type);
+
+        // if (name === 'color') console.log(state[name], name);
+
+        const typeMarkup = (
+          <span className="eui-textBreakNormal">{markup(humanizedType)}</span>
+        );
+
+        return (
+          <EuiTableRow key={name} className="playgroundKnobs__rowCell">
+            <EuiTableRowCell key={`prop__${name}-${idx}`} header="Prop">
```

Comment:
Adding the className here it's not doing anything. We want all the `EuiTableRowCell` to have a min-height of `42px` and not the `EuiTableRow`. When we add the class `EuiTableRowCell` it injects the class in a div so the min-height works. 

```suggestion
          <EuiTableRow key={name}>
            <EuiTableRowCell key={`prop__${name}-${idx}`} header="Prop" className="playgroundKnobs__rowCell">
```